keylime.conf

```diff
@@ -128,6 +128,9 @@ ek_handle = generate
 [cloud_verifier]
 #=============================================================================
 
+# Unique identifier for the each verifier instances.
+cloudverifier_id = default
```

Comment:
Is this configuration variable used by the agents or just the verifier? I know that `cloudverifier_ip` and `cloudverifier_port` are. And what happens if the value in here is different from the one that's passed to keylime_tenant?

Reply:
The `cloudverifier_id` variable is only required to configure a verifier instance. 
From the tenants POV, it is used as a parameter to request a filtered list of agents from any verifier instance.

Reply:
So the tenant reads this variable from keylime.conf? That would be confusing as it's pretty common to run tenant commands on a machine that is also running a verifier and if you start adding more verifiers it would be strange that the tenant is only talking to one of them.

Reply:
No, only the verifier reads the variable from the keylime.conf as it is defined under the `[cloud_verifier]`.